docs/guides.md

```diff
@@ -795,9 +795,69 @@ Another possible but rare problem is caused by drives that shift their identific
+
+After the upgrade has finished and the host system reboots, there may be problems with recognizing one or both of the RAID arrays. It is very unlikely that there will be a problem with the `md127` RAID 1 boot array with the most likely problem being the array operating with only one drive. Problems with the RAID 5 storage array are more likely but not common with the most likely problems being drives missing from the array or the array failing to activate. 
+
+Once the host system has rebooted, check whether the `mdadm.conf` and `dracut_mdraid.conf` files are still in the correct locations and have the correct contents. It is possible that one or both of the files have been retained; in a test upgrade from XCP-ng version 8.2 to version 8.2 on the example system, the `mdadm.conf` file was preserved as part of the upgrade while the `dracut_mdraid.conf` file was not.
+
+Missing files can be copies from the previous system by mounting the partition containing the saved copy. If not using a software RAID 1 system drive we would need to mount the second partition of the disk used as the system drive, most likely `/dev/sda2`.
+
+```
+[15:06 XCP-ng ~]# mount /dev/md127p2 /mnt
+```
+We then copy one or both files from the original system to the correct locations in the upgraded system using one or both of the commands:
+
+```
+[15:08 XCP-ng ~]# cp /mnt/etc/dracut.conf.d/dracut_mdraid.conf /etc/dracut.conf.d/
+[15:08 XCP-ng ~]# cp /mnt/etc/mdadm.conf /etc/
+```
+
+After copying the files, we unmount the original system then create a new initrd file which will contain the RAID configuration files.
+
+```
+[15:10 XCP-ng ~]# umount /mnt
+[15:10 XCP-ng ~]# dracut --force -M /boot/initrd-$(uname -r).img $(uname -r)
+```
+
+If the `mdadm.conf` or `dracut_mdraid.conf` files are damaged or cannot be copied from the backup copy of the system, it should be possible to create new copies of the files by following the instructions for creating them as part of an installation then using the above `dracut` command to create an updated initrd file.
+
+After the rebuilding of the initrd file has finished, it should be safe to reboot the host system. At this point, the system should start and run normally.
+
+In some cases it is also possible to perform an upgrade using `yum` instead of booting from CD. This type of upgrade does not completely replace the running system and does not create a backup copy. It is really a long series of updates instead of a full replacement. When upgrading a system using `yum`, the `mdadm.conf` and `dracut_mdraid.conf` files should remain in place just as with updates but copies of the files should be saved before the upgrade just in case.
```

Comment:
```suggestion
In some cases it is also possible to perform [an upgrade using `yum`](upgrade.md#from-command-line) instead of booting from CD. This type of upgrade does not completely replace the running system and does not create a backup copy. It is really a long series of updates instead of a full replacement. When upgrading a system using `yum`, the `mdadm.conf` and `dracut_mdraid.conf` files should remain in place just as with updates but copies of the files should be saved before the upgrade just in case.
```